pkg/daemon/ceph/client/subvolumegroup.go

```diff
@@ -55,3 +57,28 @@ func DeleteCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterI
 	logger.Infof("successfully deleted cephfs subvolume group %q", volName)
 	return nil
 }
+
+// PinCephFSSubVolumeGroup pin the cephfs subvolume group
+func PinCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, volName string, cephFilesystemSubVolumeGroup *cephv1.CephFilesystemSubVolumeGroup) error {
+	logger.Infof("pinning cephfs subvolume group %q of filesystem %q", cephFilesystemSubVolumeGroup.Name, volName)
+	// set default value of pinType
+	pinType := "distributed"
```

Comment:
is it fine to set default pinning values, or it should be only default for csi svg,
@Rakshith-R @batrick

Reply:
> is it fine to set default pinning values, or it should be only default for csi svg, @Rakshith-R @batrick

for all subvolumegroups. 
I've added a comment to set these defaults on the CR above.

Reply:
Updated